wls-gui-wahllokalsystem/src/views/ExampleBackendCommunicationView.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <v-container>
+    <v-col class="text-center">
+      <h2>This view shows how communication with backend-services will work</h2>
+    </v-col>
+    <v-responsive class="mx-auto">
+      <v-col class="text-center">
+        <h4>Get or Post a Broadcast message:</h4>
+        <br />
+        <v-text-field
+          class="ml-auto mr-auto"
+          width="350"
+          v-model="messageInput"
+          clearable
+          label="ID"
+        ></v-text-field>
```

Comment:
```suggestion
        <v-text-field
          v-model="messageInput"
          class="ml-auto mr-auto"
          width="350"
          clearable
          label="ID"
        ></v-text-field>
```

Reply:
siehe [5f368b4](https://github.com/it-at-m/Wahllokalsystem/pull/633/commits/5f368b41c02db539341dec60fce24cbc5cd69c0b)